addons/mail/static/src/models/attachment_viewer.js

```diff
@@ -221,15 +232,14 @@ registerModel({
         angle: attr({
             default: 0,
         }),
-        attachment: one('Attachment', {
```

Comment:
other occurrences of removed fields need to be adapted similar to the template (example `onClickDownload` still uses `attachment`)

Reply:
Oups my bad